sklearn/metrics/tests/test_common.py

```diff
@@ -789,8 +789,12 @@ def test_format_invariance_with_1d_vectors(name):
         if name not in (
             MULTIOUTPUT_METRICS | THRESHOLDED_MULTILABEL_METRICS | MULTILABELS_METRICS
         ):
-            with pytest.raises(ValueError):
-                metric(y1_row, y2_row)
+            if "roc_auc" not in name:
+                with pytest.raises(ValueError):
```

Comment:
Out of curiosity, which metrics currently hit this branch?

Reply:
Just to sanity check then, do we want to propogate the preference for a warning over an error to any of these as well?